src/backend/marsha/core/models/video.py

```diff
@@ -402,4 +402,72 @@ def get_source_s3_key(self, stamp=None):
 
         """
         stamp = stamp or to_timestamp(self.uploaded_on)
-        return f"{self.video.pk}/thumbnail/{self.pk}/{stamp}"
+        return "{video!s}/thumbnail/{pk!s}/{stamp:s}".format(
+            video=self.video.pk, pk=self.pk, stamp=stamp
+        )
+
+
+class LiveRegistration(BaseModel):
+    """Model representing a scheduling live registration."""
+
+    RESOURCE_NAME = "liveregistrations"
+
+    email = models.EmailField(_("email address"), db_index=True)
+
+    consumer_site = models.ForeignKey(
+        blank=True,
+        help_text=_("Only present for lti users."),
+        null=True,
+        on_delete=models.CASCADE,
+        related_name="liveregistrations",
+        to="ConsumerSite",
+        verbose_name=_("LTI consumer site"),
+    )
+
+    lti_user_id = models.CharField(
+        blank=True,
+        db_index=True,
+        help_text=_(
+            "Unique identifier for the user on the tool consumer only present for lti users"
```

Comment:
I thin punctuation is missing. `Unique identifier for the user on the tool consumer. Only present for lti users` maybe something like this ?